client/unit-tests/coretypes/test_filtering.py

```diff
@@ -56,17 +65,18 @@ def test__format_filter(geojson):
         },
     )
 
-    filter_from_constraints = _format_filter(
+    filter_from_constraints = Filter.create(
         [
-            Dataset.name.in_(["a", "b", "c"]),  # type: ignore - filter type error
-            Model.name.in_(["x", "y", "z"]),  # type: ignore - filter type error
-            Label.score > 0.75,  # type: ignore - filter type error
-            Annotation.polygon.area > 1000,  # type: ignore - filter type error
-            Annotation.polygon.area < 5000,  # type: ignore - filter type error
-            Dataset.metadata["some_str"] == "foobar",  # type: ignore - filter type error
-            Dataset.metadata["some_float"] >= 0.123,  # type: ignore - filter type error
-            Dataset.metadata["some_datetime"] > datetime.timedelta(days=1),  # type: ignore - filter type error
-            Dataset.metadata["some_geospatial"].intersect(geojson),  # type: ignore - filter type error
+            Dataset.name.in_(["a", "b"]) | (Dataset.name == "c"),
+            (Model.name == "x") | Model.name.in_(["y", "z"]),
```

Comment:
could this be written as `Model.name.in_(["x", "y", "z"])`? from these tests, it looks like `.in_` may only accept a list of two elements.. just checking my understanding here

Reply:
This could most definitely be written as `Model.name.in_(["x", "y", "z"])`

If you compare the highlighted line with the next, my intent was to show that the OR operation was commutative.

I can change back to the 3-arg `in_` if you think this is confusing.

Reply:
gotcha, very cool! could you change either line 70 or line 71 back to show the three-way `in_`? I think that would make it clear that other operation is acceptable